src/sidebar/util/service-context.js

```diff
@@ -103,6 +107,7 @@ export function withServices(Component) {
  * context of custom hooks.
  *
  * @param {string} service - Name of the service to look up
+ * @return {Store}
```

Comment:
`useService` doesn't _always_ return a Redux store. It returns the named service, so the type depends on the service name. The easiest option, since we don't use it in a lot of places, would be to just `@return {Object}` and then cast it where you're using it.